iocore/net/quic/QUICUnidirectionalStream.cc

```diff
@@ -401,7 +403,7 @@ QUICSendStream::largest_offset_sent() const
 //
 QUICReceiveStream::QUICReceiveStream(QUICRTTProvider *rtt_provider, QUICConnectionInfoProvider *cinfo, QUICStreamId sid,
                                      uint64_t recv_max_stream_data)
-  : QUICStreamVConnection(cinfo, sid),
+  : QUICStreamVConnection(cinfo, sid, QUICStreamType::RECEIVE_ONLY),
```

Comment:
Why do you want to pass RECEIVE_ONLY? You can determine it by reading a stream id.

Reply:
Passing send/receive/both would make sense if we had a stream factory that internally assigns a stream id.

For example:
```cpp
QUICStream *stream = stream_factory.create(RECEIVE);
---
QUICStream *
QUICStreamFactory::create(QUICStreamDicrection direction) {
  QUICStream stream;

  QUICStreamId id = this->_get_next_stream_id(direction);
  switch (direction) {
  case RECEIVE:
    stream = new QUICReceiveStream(this->cinfo, id);
  }
  return stream;
}
```

Reply:
That make sense. will do it.

Reply:
update ! can you please review again.